app/static/lang/vi-VI.js

```diff
@@ -78,6 +78,12 @@ export default () => {
         wallets: 'Ví',
         title: 'Danh  những ví Lotus đang khả dụng trong hệ thống:',
       },
+      banner: {
+        why_1: 'Xã hội thường gặp khó khăn trong việc cấp vốn cho các dự án công cộng - việc phân bổ nguồn lực vào những công việc cấp bách nhưng không có lợi ích kinh tế. Với vấn đề này, Lotus đá được tạo ra. Chúng tôi tin tưởng rầng bằng cách thay đổi cách nhìn nhận về đồng tiền, và thách thức những sáng kiến xung quanh việc sử dụng đồng tiền, chúng tôi có thể giải quyết được việc cấp vốn cho những dự án công cộng.',
+        why_2: 'Mục tiêu của chúng tôi là để bày tỏ lòng tri ân với các tình nguyện viên của các dự án từ thiện bằng cách gửi Lotus cho họ một cách miễn phí. Và, như là một cư xử văn hóa, sẵn lòng chấp nhận những đồng tiền này để đổi lại những thứ mà những tình nguyện viên này cần.',
+        why_3: 'Những tập trung đầu tiên của chúng tôi là các dự án mã nguồn mỡ, và đặc biệt là các dự án thiết yếu để giúp Lotus tồn tại và phát triển. Chúng tôi sẵn lòng để thảo luận với các dự án tự nguyện mong muốn tham gia. Chúng tôi đặc biệt quan tâm cho việc cấp phát vốn cho các dự án mã nguồn mở, trường học, trại mồ côi, bác sỹ, và các tiện ích công cộng khác.',
```

Comment:
Những tập trung đầu tiên của chúng tôi là các dự án mã nguồn mỡ --> change to "Những tập trung đầu tiên của chúng tôi là các dự án mã nguồn mở"

Reply:
Lotus đá được tạo ra --> Change to "Lotus đã được tạo ra" on Why_1

Reply:
Here the full text updates:

Why_1
'Xã hội thường gặp khó khăn trong việc cấp vốn cho các dự án công cộng - việc phân bổ nguồn lực vào những công việc cấp bách nhưng không có lợi ích kinh tế. Với vấn đề này, Lotus đã được tạo ra. Chúng tôi tin tưởng rầng bằng cách thay đổi cách nhìn nhận về đồng tiền, và thách thức những sáng kiến xung quanh việc sử dụng đồng tiền, chúng tôi có thể giải quyết được việc cấp vốn cho những dự án công cộng.'

why_3: 'Những tập trung đầu tiên của chúng tôi là các dự án mã nguồn mở, và đặc biệt là các dự án thiết yếu để giúp Lotus tồn tại và phát triển. Chúng tôi sẵn lòng để thảo luận với các dự án tự nguyện mong muốn tham gia. Chúng tôi đặc biệt quan tâm cho việc cấp phát vốn cho các dự án mã nguồn mở, trường học, trại mồ côi, bác sỹ, và các tiện ích công cộng khác.',

Reply:
Can use github suggestions to make it easy to just merge the change here.